src/test/kotlin/com/ampnet/walletservice/service/TransactionInfoServiceTest.kt

```diff
@@ -25,7 +25,9 @@ class TransactionInfoServiceTest : JpaServiceTestBase() {
     fun mustCreateOrgTransaction() {
         suppose("Service can create org transactionInfo") {
             testContext.transactionInfo =
-                transactionInfoService.createOrgTransaction(organizationUuid, testContext.organizationName, userUuid)
+                transactionInfoService.createOrgTransaction(
+                    organizationUuid, testContext.organizationName, createUserPrincipal(userUuid, coop = coop)
```

Comment:
`coop` is always the same value "ampnet" so I don't see the purpose of setting it to the same value as default value in function `createUserPrincipal`